my-react-app/src/app/screens/Game/components/Board/index.js

```diff
@@ -29,4 +30,9 @@ class Board extends React.Component {
   }
 }
 
+Board.propTypes = {
+  squares: PropTypes.array,
+  onClick: PropTypes.func
```

Comment:
`onClick` is required in this case: if the person using `Board` didn´t pass an `onClick` prop, the component wouldn´t work